packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -201,6 +201,17 @@ export function computeSize(entry: RumPerformanceResourceTiming) {
   return undefined
 }
 
+export function computeDetailedSizes(entry: RumPerformanceResourceTiming) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Instead of introducing a new function you could update `computeSize` to add the new fields. This would avoid duplicating the check `entry.startTime < entry.responseStart`. Wdyt?